core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -160,9 +152,9 @@ object ConsumerGroupCommand {
     }
 
     protected def describeGroup(group: String) {
-      val configs = parseConfigs(opts)
```

Comment:
It was @junrao suggestion and I agree that to have both commandConfigOpt and configOpt makes usage a bit confusing. I am fine with reverting this, however I will wait for @junrao to pitch in his thoughts.